### ISSUE TITLE
error loading model from lm studio

### ISSUE THREAD

**Issue by michel-io** [devoxx/DevoxxGenieIDEAPlugin#303]
I'm running lm studio v0.3.2 as a local server

the plugin connects flawlessly to the server, and the server send the list of models loaded, but the plugin never show the list of models available

**Comment by stephanj** [comment]
Will have a closer look once Devoxx Belgium is behind us :)

**Comment by mydeveloperplanet** [comment]
I think this one is fixed in v0.2.24. I have two models loaded (console logs LMStudio):
```
2024-11-03 17:15:37  [INFO]
Returning {
  "data": [
    {
      "id": "qwen2.5-coder-7b-instruct",
      "object": "model",
      "owned_by": "organization_owner"
    },
    {
      "id": "llama-3.2-1b-instruct",
      "object": "model",
      "owned_by": "organization_owner"
    }
  ],
  "object": "list"
}
```

Both models are shown in DevoxxGenie
![image](https://github.com/user-attachments/assets/73eddb7a-0311-41ae-9339-c71307405110)

I can use both models.

@michel-io  can you verify this?
